Sources/ArcGISToolkit/Components/Augmented Reality/ARSwiftUIView.swift

```diff
@@ -153,3 +154,40 @@ class ARSwiftUIViewProxy: NSObject, ARSessionProviding {
         )
     }
 }
+
+extension ARSwiftUIViewProxy {
+    /// Performs a raycast to get the transformation matrix representing the corresponding 
+    /// real-world point for `screenPoint`.
+    /// - Parameter screenPoint: The screen point to determine the real world transformation matrix from.
+    /// - Returns: A `TransformationMatrix` representing the real-world point corresponding to `screenPoint`.
+    func raycast(from screenPoint: CGPoint) -> TransformationMatrix? {
+        // Use the `raycastQuery` method on ARSCNView to get the location of `screenPoint`.
+        guard let query = raycastQuery(
+            from: screenPoint,
+            allowing: .existingPlaneGeometry,
```

Comment:
https://developer.apple.com/documentation/arkit/arraycastquery/target/estimatedplane

According to this doc, maybe we should use `estimatedPlane` for real world, and `existingPlaneGeometry` for table top.